_sass/blocks/_revenues_subpage_intro.scss

```diff
@@ -43,9 +69,16 @@
   }
 
   padding-left: $base-padding;
-  padding-bottom: $base-padding;
+  padding-bottom: 0;
   padding-right: $base-padding-lite;
 
+  strong {
+    font-weight: $weight-book;
+  }
+}
+.revenues_subpage-office_text,
```

Comment:
Selector `revenues_subpage-office_text` should be written in lowercase with hyphens